### ISSUE TITLE
Pisugar-server 'Invalid'

### ISSUE THREAD

**Issue by Zanop** [PiSugar/PiSugar#154]
Hi,

Looking at syslog I am seeing this constantly being printed:

Nov 13 13:30:27 Curigotchi pisugar-server[243]: [2024-11-13T12:30:27Z ERROR pisugar_server] Invalid
Nov 13 13:30:41 Curigotchi pisugar-server[243]: [2024-11-13T12:30:41Z ERROR pisugar_server] Invalid
Nov 13 13:30:53 Curigotchi pisugar-server[243]: [2024-11-13T12:30:53Z ERROR pisugar_server] Invalid
Nov 13 13:31:04 Curigotchi pisugar-server[243]: [2024-11-13T12:31:04Z ERROR pisugar_server] Invalid
Nov 13 13:31:15 Curigotchi pisugar-server[243]: [2024-11-13T12:31:15Z ERROR pisugar_server] Invalid

No further information is seen as to what is invalid.

Seems to be casued by my browser being open at 'http://10.0.0.2:8421/#/". 

Regards,
Vlado

**Comment by JdaieLin** [comment]
Does the webUI work properly?

**Comment by Zanop** [comment]
I don’t see errors in the ui

**Comment by JdaieLin** [comment]
Yes the webUI will keep querying the server. The server will resonse "Invalid" if there's an invalid command. I'm not sure which query is causing this, maybe it's dereapted in the server but still left in the UI. I will check it later.

**Comment by JdaieLin** [comment]
> Hi,
> 
> Looking at syslog I am seeing this constantly being printed:
> 
> Nov 13 13:30:27 Curigotchi pisugar-server[243]: [2024-11-13T12:30:27Z ERROR pisugar_server] Invalid Nov 13 13:30:41 Curigotchi pisugar-server[243]: [2024-11-13T12:30:41Z ERROR pisugar_server] Invalid Nov 13 13:30:53 Curigotchi pisugar-server[243]: [2024-11-13T12:30:53Z ERROR pisugar_server] Invalid Nov 13 13:31:04 Curigotchi pisugar-server[243]: [2024-11-13T12:31:04Z ERROR pisugar_server] Invalid Nov 13 13:31:15 Curigotchi pisugar-server[243]: [2024-11-13T12:31:15Z ERROR pisugar_server] Invalid
> 
> No further information is seen as to what is invalid.
> 
> Seems to be casued by my browser being open at 'http://10.0.0.2:8421/#/".
> 
> Regards, Vlado

Which model are you using, PiSugar 2 or 3?

**Comment by JdaieLin** [comment]
Try set up a schedual wake up time and see if the 'Invalid' log comes up again.
